hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieHiveCatalog.java

```diff
@@ -96,12 +96,17 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import static org.apache.flink.table.factories.FactoryUtil.CONNECTOR;
+import static org.apache.flink.util.Preconditions.checkArgument;
+import static org.apache.flink.util.Preconditions.checkNotNull;
```

Comment:
Please fix the indentation, the `org.apache.hudi.` has the supreme priority.

Reply:
hi, putting org.apache.flink in front is to solve the checkstyle problem

<img width="1296" alt="image" src="https://github.com/apache/hudi/assets/28925346/866be6fd-86e3-49a2-8258-325bfea418f7">